README.md

```diff
@@ -330,6 +331,9 @@ Focus is on the specific task.
 - [Rocketeer](https://github.com/Anahkiasen/rocketeer) - Deployment
 - [Travis CI](https://travis-ci.org/) - A continuous integration platform - de-facto standard for testing (GitHub) repos.
 
+## Feature Toggle
+- [Swivel](https://github.com/zumba/swivel-cake) - A library that allows to setup up to 10 different behaviors with an interface for tracking results.
```

Comment:
@jrbasso I think @dereuromark's inquiry was due to the phrasing "A library that...".  So perhaps rephrase the description to mention it's a plugin which wraps thats library.

Also would be better if you could put it one of the existing categories instead of a new one. There are already too many :)